src/main/java/org/takes/misc/Utf8String.java

```diff
@@ -74,4 +75,12 @@ public Utf8String(final byte... bytes) {
     public String string() {
         return this.value;
     }
+
+    /**
+     * Returns the english lower case string representation.
+     * @return String value
+     */
+    public String englishLowerCase() {
+        return this.value.toLowerCase(Locale.ENGLISH);
```

Comment:
@dalifreire I have a concern here - this method does not depend on string being UTF8 or not, so it doesn't fit well this class I think